components/admin_console/admin_definition.jsx

```diff
@@ -2972,6 +2972,369 @@ export default {
+                            label_default: 'Email Batching Interval',
+                            help_text: t('admin.experimental.emailBatchingInterval.desc'),
+                            help_text_default: 'Specify the maximum frequency, in seconds, which the batching job checks for new notifications. Longer batching intervals will increase performance.',
+                            help_text_markdown: false,
+                            placeholder: t('admin.experimental.emailBatchingInterval.example'),
+                            placeholder_default: 'E.g.: "30"',
+                        },
+                        // {
+                        //     type: Constants.SettingsTypes.TYPE_TEXT,
+                        //     key: 'EmailSettings.ReplyToAddress',
+                        //     label: t('admin.experimental.replyToAddress.title'),
+                        //     label_default: 'Reply To Address',
+                        //     help_text: t('admin.experimental.replyToAddress.desc'),
+                        //     help_text_default: 'TODO',
+                        //     help_text_markdown: true,
+                        //     placeholder: t('admin.experimental.replyToAddress.example'),
+                        //     placeholder_default: 'E.g.: "reply-to@example.com"',
+                        // },
+                        {
+                            type: Constants.SettingsTypes.TYPE_BOOL,
+                            key: 'ThemeSettings.EnableThemeSelection',
+                            label: t('admin.experimental.enableThemeSelection.title'),
+                            label_default: 'Enable Theme Selection',
+                            help_text: t('admin.experimental.enableThemeSelection.desc'),
+                            help_text_default: 'Enables the **Display > Theme** tab in Account Settings so users can select their theme.',
+                            help_text_markdown: true,
+                            isHidden: needsUtils.not(needsUtils.hasLicense), // E10 and higher
+                        },
+                        {
+                            type: Constants.SettingsTypes.TYPE_BOOL,
+                            key: 'ThemeSettings.AllowCustomThemes',
+                            label: t('admin.experimental.allowCustomThemes.title'),
+                            label_default: 'Allow Custom Themes',
+                            help_text: t('admin.experimental.allowCustomThemes.desc'),
+                            help_text_default: 'Enables the **Display > Theme > Custom Theme** section in Account Settings.',
+                            help_text_markdown: true,
+                            isHidden: needsUtils.not(needsUtils.hasLicense), // E10 and higher
+                            isDisabled: needsUtils.stateValueFalse('ThemeSettings.EnableThemeSelection'),
+                        },
+                        // {
```

Comment:
The server needs an actual array (vs. a comma-separated list), so I'm deferring this one for now.